libr/util/strbuf.c

```diff
@@ -73,7 +73,14 @@ R_API bool r_strbuf_reserve(RStrBuf *sb, int len) {
 
 R_API bool r_strbuf_setbin(RStrBuf *sb, const ut8 *s, int l) {
 	r_return_val_if_fail (sb && s, false);
-	r_return_val_if_fail (l >= 0, false);
+	if (l < 0) {
```

Comment:
why do we want to support this thing? If you are using `r_strbuf_setbin`, IMHO, you *must* pass a valid size as well. -1 would assume that `s` is a string, but if you know that you may as well directly call `r_strbuf_set` instead of this.

So, IMHO this can be simplified by making `l` a `size_t` and assuming `l` is always valid.